ql/src/test/org/apache/hadoop/hive/ql/exec/PartitionUtil.java

```diff
@@ -82,4 +88,26 @@ public static Set<CheckResult.PartitionResult> createPartsNotInMs(int numOfParts
         }
         return partsNotInMs;
     }
+
+
+    public static void addPartitions(IMetaStoreClient db, String dbName, String tableName, String location,
+         HiveConf hiveConf) throws Exception {
+        List<Partition> partitions = new ArrayList<>();
+        for (int i = 0; i < 30; i++) {
```

Comment:
Parameterize the partition count to make it more usable

Reply:
changed